src/components/UserActivityPage/UserActivityPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useMemo } from 'react';
+import { nullable } from '@taskany/bricks';
+import { Button, Text } from '@taskany/bricks/harmony';
+
+import { TrpcRouterOutput, trpc } from '../../trpc/trpcClient';
+import { HistoryRecord } from '../HistoryRecord/HistoryRecord';
+import { LayoutMain } from '../LayoutMain';
+import { HistoryEventData } from '../../modules/historyEventTypes';
+
+import s from './UserActivityPage.module.css';
+import { tr } from './UserActivityPage.i18n';
+
+interface UserActivityPageProps {
+    userId: string;
+}
+
+const UserActivityPageInner = ({
+    user,
+    events,
+    loadMore,
+    hasNext,
+}: {
+    user: TrpcRouterOutput['user']['getById'];
+    events: TrpcRouterOutput['historyEvent']['getUserActivity']['events'];
+    loadMore: VoidFunction;
+    hasNext?: boolean;
+}) => {
+    const actingUser = useMemo(() => ({ id: user.id, name: user.name, email: user.email }), [user]);
```

Comment:
for what?

why not `actingUser: user` on line 38 ?

Reply:
Excess data was very confusing during development and debugging

Reply:
But you are right, i fixed it